Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+
+\section{Benefit to SPEX}
+\label{sec:benefit}
+% One of the core values of SPEX is to provide opportunities for academic and professional growth for its members,
+% and to challenge them with interesting projects.
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+
+This project would be benefical to SPEX because of the pathway it creates. After its completion, SPEX will have a cold gas thruster that
+be shown to companies, faculty, students, and at any event where SPEX is on display. The CGT's high mobility will make transportation easy, and since
+the system is heavily dependent on chamber pressure for thrust, it is very safe to show to people when the tank is unpressurized. This project
+will also be the first foot in the door for CubeSat propulsion. There is also a continuation of this project ready to be initated upon its completion, the
+reaction control system.
+
+There are many levels of
+analysis that can be used to design one, but the best way is experimentation (trial \& error). This is especially true for cold gas propulsion of this
+size. The combination of the size of the nozzle and the high reynolds number of the gas does not allow and analytical solution to the optimal dimensions
+of the nozzle. An analytical solution usually concerns making a conical nozzle. However the thrust loss on these nozzles is maxmized. This is due to the
+direction of the outgoing flow. The particles travel away from the nozzle at an angle and this increases the horizontal velocity component, and decreases the
+vertical velocity (the direction pointing out of the nozzle), which decreases the thrust of the system. Since cold gas propulsion is inherently inefficient, due
+to the fact that their is no chemcial reaction or heat applied to the system, maxmizing the thrust is imperitive.
+
+Last, but certainly not least, is the nozzle determination experience. Because of the flow conditions and dimensions, an analytical solution will not be sufficient. To accurately determine
+the nozzle size experiementation must happen. This data will be immensely helpful for designing the RCS nozzles and any other CGT nozzles with similar
+size magnitudes. To do this the team will operate at several different inlet pressures and determine the correct size of these nozzles to create optimal expansion.
+The team will then plot this data and fit a polynomial approximation to it. This polynomial will serve as a initial guess for future nozzles that will be
+expanded upon and optimized during testing.
+
+% Below I have used subsections to identify key ideas in this section. These particular subsections are not required as part of the SPEX Standard, but serve as an example of using subsections in a text.
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+This project will include a lot of design and data analysis. Once the design phase
```

Comment:
This is assumed. Instead, clearly define how much and how long the design phase will be.